scalatags/shared/src/test/scala/scalatags/text/PrettyTest.scala

```diff
@@ -0,0 +1,18 @@
+package scalatags.text
+import utest._
+
+object PrettyTests extends TestSuite{
+
+  import scalatags.Text.all._
+  import scalatags.text.Utils.Pretty
+
+  def tests = TestSuite{
+
+    'helloWorld{
+      val sample = div("omg")
+      assert(sample.pretty.toString == "<div>\n"+
```

Comment:
This could be a triple-quoted string, and can probably go in one of the existing files e.g. `ExampleTests` rather than being a whole new file